_data/simple-icons.json

```
@@ -10379,6 +10379,11 @@
             "hex": "000000",
             "source": "https://near.org/brand"
         },
+        {
+            "title": "Nebula",
+            "hex": "000000",
```

```suggestion
            "hex": "2CADFE",
```

They seem to use this color as the accent/primary across the main website.